akka-remote/src/main/scala/akka/remote/artery/compress/OutboundActorRefCompression.scala

```diff
@@ -101,6 +107,39 @@ private[remote] class OutboundCompressionTable[T](system: ActorSystem, remoteAdd
 
 }
 
+class HashScanIndexMap[T](val size: Int) {
```

Comment:
Had random thoughts about even more specialized versions hm... Will have some fun in the evening with them